python/dgl/distributed/rpc_server.py

```diff
@@ -2,9 +2,9 @@
 
 from . import rpc
 from .constants import MAX_QUEUE_SIZE
-from .server_state import get_server_state
+from .server_state import ServerState
 
-def start_server(server_id, ip_config, num_clients, \
+def start_server(server_id, ip_config, num_clients, server_state, \
```

Comment:
why not just call get_server_state to get server state?

Reply:
get_server_state() is a c API but here we didn't use any c code.